spokestack/config.py

```diff
@@ -0,0 +1,12 @@
+"""
+This module contains the SpeechConfig object which
+sets the parameters for the SpeechPipeline
+"""
+
+
+class SpeechConfig:
+    """ Speech Configuration """
+
+    def __init__(self, config: dict) -> None:
```

Comment:
configuration dictionaries make sense in the java world, but they aren't very pythonic
it would make more sense to use kwargs on the SpeechConfig class to set up the configuration properties
another option would be to get rid of SpeechConfig altogether and use kwargs on the constructors of the components that actually need configuration values